p2p/discover/v4_udp.go

```diff
@@ -63,15 +65,15 @@ const (
 
 // RPC packet types
 const (
-	pingPacket = iota + 1 // zero is 'reserved'
-	pongPacket
-	findnodePacket
-	neighborsPacket
+	p_pingV4 = iota + 1 // zero is 'reserved'
```

Comment:
Nit: Consider naming this `pingV4Packet`. I'm not sure how I feel about the `p_` prefix.

Reply:
KRISHNAN .P MERCHANT EXPORTER BRL EXPORTS web www brlexports.com